src/helpers/contentWrapper.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react'
+import { sortBy } from 'lodash'
+import { useQuery } from 'cozy-client'
+import { updateIndexFullNameAndDisplayName } from '../helpers/contacts'
+
+/**
+ * Get contacts with indexes,
+ * update contacts without indexes and
+ * sort them by indexes
+ * @param {array} contactsWithIndexes - contacts with indexes
+ * @param {array} contactsWithNoIndexes - contacts without indexes
+ * @returns {array} contacts with indexes and sorted
+ */
+export const updateAndSortByFamilyNameGivenNameEmailCozyUrl = (
```

Comment:
Nit: the `update` in the name led me to think the function would perform server-side updates. IMO something like `getContactsSortedBy[etc]` might be clearer.

Reply:
Suite à discussion récente, `get` quant à lui fait penser à une synchronisation.

Que penses-tu de `reworkAndSortByFamilyNameGivenNameEmailCozyUrl` ?

Reply:
C'est pas mal, sinon `consolidateAnd…`? Ou `harmonizeAnd…`? Mais rework me va aussi, un truc dans le genre quoi 👍

Reply:
Va pour `harmonizeAnd`

Reply:
When we transform the data to conform to a given schema or to make sure that is coherent with other documents, I like to use `normalize`. This is also what is used in the `cozy-client` codebase to make sure that data from the server is consistent between each call.

I would reserve "harmonize" for cases where the each item of the data needs to be tweaked a bit to work well with other items. Here, the normilization denotes that each datum is tweaked individually.

The fact that the function needs a "And" in its name is a smell IMHO. It indicates that the function is doing too much. It should be clearer if the caller does one function call to normalize and one function call to sort.

The name of the function here is a bit too telling also I think, the sort key is a detail of implementation that should not leak to the caller.